pages.pt_BR/common/2to3.md

```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversão automática de código Python 2 para Python 3.
+> Mais informação: <https://docs.python.org/3/library/2to3.html>.
```

Comment:
We normally use `Mais informações` for this label.  Any reason for doing it differently?
```suggestion
> Mais informações: <https://docs.python.org/3/library/2to3.html>.
```

Reply:
Now I read it. We use a plural like informations. 🤔